src/sentry/api/event_search.py

```diff
@@ -635,11 +635,17 @@ def _handle_basic_filter(self, search_key, operator, search_value):
         # If a date or numeric key gets down to the basic filter, then it means
         # that the value wasn't in a valid format, so raise here.
         if search_key.name in self.date_keys:
-            raise InvalidSearchQuery(f"{search_key.name}: Invalid format for date field")
+            raise InvalidSearchQuery(
+                f"{search_key.name}: Invalid date: {search_value.raw_value}. Expected +/-duration (e.g. +1h) or ISO 8601 (e.g. {datetime.now().isoformat()[:-4]})."
```

Comment:
Is this usually RFC 3339?

Reply:
ok so after some research, our date format is a subset of RFC 3339 and ISO 8601.